test_utils.js

```diff
@@ -51,6 +51,7 @@ const DEFAULT_GLOBAL_CONFIG: GlobalConfig = {
   verbose: false,
   watch: false,
   watchAll: false,
+  watchPlugins: [],
```

Comment:
With Jest Runners we have the problem that they are hard to configure... This is mainly because there is no way to pass options to runners, I wonder if we would have a similar problem with plugins

Proposed solution for customizing runners: #4278 
Example of how existing runners need to do configuration right now:  https://github.com/jest-community/jest-runner-eslint#options

I really like the idea of using an array for customization
- without options: `watchPlugins: ['my-plugin']`
- with options: `watchPlugins: [['my-plugin', { someOptions: true, other: 'foo' }]]`

Reply:
^ I like that, yes.